course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
+1. Why is it beneficial to use a programming language rather than a spreadsheet software for data analyses?
+2. Get setup with Rstudio at home (or a lab computer after hours). Install R, Rstudio, and the necessary packages (tidyverse, rmarkdown). Use `install.packages(<package_name>)`. Load all the libraries you installed into your environment with `library(<package_name>)` and send us the output of running `sessionInfo()`.
```

Comment:
Good point, I forgot about that LaTeX needs to be installed... I haven't done it on Windows myself, but it [seems straightforward](https://medium.com/@sorenlind/create-pdf-reports-using-r-r-markdown-latex-and-knitr-on-windows-10-952b0c48bfa9), just install MikTeX as you mentioned.

I will follow up in #50 and see if we can get these installed. I will try it first to see which R-latex packages are needed. I think this is important since a strong point for me to use markdown in the first place is that I can easily render PDFs and hand in for any class assignment. For this purpose, it is not as useful to teach how to generate markdown documents, since teachers will probably not accept text files for assignments.